tests/test-cases/Group6-VIC-Machine/6-13-TLS.robot

```diff
@@ -32,7 +32,27 @@ Create VCH - defaults with --no-tls
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
+Create VCH - defaults custom cert path
+    Set Test Environment Variables
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    ${pwd}=     Run  pwd
```

Comment:
curdir is the absolute path to the directory the test file is located in and execdir is the absolute path to the directory that execution of the script occurred in